lib/node_modules/@stdlib/string/next-grapheme-cluster-break/README.md

```diff
@@ -0,0 +1,174 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2020 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# nextGraphemeClusterBreak
+
+> Return the next extended grapheme cluster break in a string after a specified position.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var nextGraphemeClusterBreak = require( '@stdlib/string/next-grapheme-cluster-break' );
+```
+
+#### nextGraphemeClusterBreak( string, position )
```

Comment:
Any particular reason to make `position` a required parameter, rather than optional parameter as in the original [RFC](https://github.com/stdlib-js/stdlib/issues/357)? I would think that a reasonable default `fromIndex` would be `0`.

Reply:
I did this because it seemed quite natural that `nextGraphemeClusterBreak` would need a position to justify the "next". Also, does this mean that `prevtGraphemeClusterBreak` would have to have a default `beforeIndex` of `str.length - 1`?

Reply:
@ShraddheyaS Yeah, I sympathize with the conflict with `next` and `fromIndex`; however, I think that, for consistency with other similar functions, such as `indexOf`, we should have the second argument be optional.

Re: `prev*`. This is a tricky one, where I don't think there is a single "right" answer. There would be nothing "wrong" with iterating a string from start to end and invoking this API, in which case a default of `0` would be fine. Alternatively, if we believe that `prev*` is intended for end to start iteration, then `str.length-1` makes sense. I think it is fine to go with `str.length-1`.

Reply:
Yes, `prev*` is intended for an end to start iteration. Will make the defaults as `0` and `str.length - 1` for `next*` and `prev*` respectively.

Reply:
Sounds good!